src/components/Availability.js

```diff
@@ -36,8 +37,17 @@ export default function Availability({ entry }) {
                 </div>
             );
         } else {
+            const totalAvailableSlots = availableSlots.reduce(
+                (total, slot) => total + slot.numberAvailableAppointments,
+                0
+            );
             return (
                 <div>
+                    {totalAvailableSlots > 1 && availableSlots.length > 1 && (
+                        <Typography variant="h6">
```

Comment:
For accessibility reasons, we want to avoid skipping header levels. Unless there is an `h5` somewhere higher up the page, we should change this. If you need the `h6` default styles, create a class that imitates this.

Reply:
Hey @ramon-h , actually UX wants it just bold so you can remove the heading :)

Reply:
Just pushed the change. I had noticed the usage of Material but not `makeStyles`.